app/src/main/java/com/framgia/englishconversation/screen/audiodetail/AudioDetailViewModel.java

```diff
@@ -80,6 +82,13 @@ public void onItemUserNameClick(TimelineModel item) {
                     mActivity.startActivity(
                             ProfileUserActivity.getInstance(mActivity, item.getCreatedUser()));
                 }
+
+                @Override
+                public boolean onItemLongClick(View viewGroup, TimelineModel item) {
+                       return false;
+                }
```

Comment:
ERROR: 'method def rcurly' have incorrect indentation level 16, expected level should be one of the following: 8, 12.